src/Options/ContactDetails.php

```diff
@@ -0,0 +1,137 @@
+<?php namespace Leean\Elements\Options;
+
+/**
+ * Class ContactDetails.
+ *
+ * @package Leean\Elements\Options
+ */
+class ContactDetails
```

Comment:
Hi @mitogh 

One of the collections of options is ContactDetails
Another is SocialLinks

If I change the namespace, it doesn't make much sense to have Contact\Details and Social\Links.
The are both collections of options and I think they should be in the same namespace.

I did try to change it anyway, but I couldnt get it to work either.